passkeys/backend.py

```diff
@@ -1,16 +1,18 @@
 from django.contrib.auth.backends import ModelBackend
-
+from .FIDO2 import auth_complete
 
 class PasskeyModelBackend(ModelBackend):
     def authenticate(self, request, username='',password='', **kwargs):
+
         if request is None:
             raise Exception('request is required for passkeys.backend.PasskeyModelBackend')
+
         if username!='' and password != '':
             request.session["passkey"]={'passkey':False}
             return super().authenticate(request,username=username,password=password, **kwargs)
-        passkeys = request.POST.get('passkeys')
-        if not passkeys:
+
+        passkeys = request.POST.get('passkeys', None)
```

Comment:
This is a validation check that the application is setup properly, so why?

Reply:
Because you may receive an empty string in request.POST.get('passkeys'), so it makes sense to validate that it is neither None nor an empty string. If you receive no value, the default is None, which will result in an error.

```
passkeys = request.POST.get('passkeys', None)
if passkeys is None or passkeys == '':
```

Reply:
Empty string is acceptable value as it means that the passkeys field is present in the form. Empty passkeys field means that the user didn't use passkeys which is acceptable. So it doesn't need to raise an exception

Reply:
Yes, it makes perfect sense!